packages/Code/src/WindowChrome.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { cx, css } from '@leafygreen-ui/emotion';
+import { Variant, variantColors } from '@leafygreen-ui/syntax';
+import { darken } from 'polished';
+
+export const windowChromeHeight = 28;
+
+const windowChromeStyle = css`
+  height: ${windowChromeHeight}px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const windowControlsStyle = css`
+  position: absolute;
+  left: 12px;
+  top: 0;
+  bottom: 0;
+  height: 12px;
+  margin: auto;
+  display: flex;
+`;
+
+function WindowControl({ color }: { color: string }) {
+  return (
+    <div
+      className={css`
```

Comment:
this feels like it should be a className defined outside the component

Reply:
The reason I inlined this style block is due to the dynamic styles included. Having the className defined outside of the component would've required splitting the styles into two composable style strings which seemed far less clean.

Usually I'd be inclined to not have this defined inline, but in this situation it seemed best.